api/openapi.yaml

```diff
@@ -1612,6 +1622,63 @@ components:
       #  - target
       #  - scan
 
+    TargetScanResultRelationship:
```

Comment:
maybe a comment here and in `TargetScanResult` that if one the field is modified the other object should be modified accordantly? This are basically the same objects.